apps/costcontrol/test/unit/formatting_test.js

```diff
@@ -134,44 +134,17 @@ suite('Formatting Test Suite >', function() {
   test(
     'formatTime() test behaviour.',
     function() {
-      var originalMozHour12 = window.navigator.mozHour12;
-
       var f = Formatting;
       var today = Toolkit.toMidnight(new Date());
       var MINUTE = 60 * 1000;
 
       var yesterday = Toolkit.toMidnight(new Date(today.getTime() - MINUTE));
       var oneTuesday = Toolkit.toMidnight(new Date(2014, 1, 25));
-      // As navigator.mozl10n.DateTimeFormat is a constructor, it is not
-      // possible to put an spy on its method localeFormat. For this reason,
-      // the spy must be placed into the Mock, instead of in the real method.
-      var dateSpy = this.sinon.spy(MockMozL10n.DateTimeFormat.prototype,
-                        'localeFormat');
 
       assert.equal(f.formatTime(), 'never');
-      f.formatTime(oneTuesday, '%a');
-      assert.ok(dateSpy.calledWith(oneTuesday, '%a'));
-      // formatTime method does not call localeFormat method when the formatted
-      // date is yesterday or today, in that case, uses location tags.
-      assert.ok(f.formatTime(today).contains('today'));
-      assert.ok(f.formatTime(yesterday).contains('yesterday, '));
-
-      dateSpy.reset();
-      window.navigator.mozHour12 = true;
-      f.formatTime(oneTuesday);
-      assert.ok(dateSpy.calledTwice);
-      assert.ok(dateSpy.calledWith(oneTuesday, '%a'));
-      assert.ok(dateSpy.calledWith(oneTuesday,  'shortTimeFormat12'));
-
-      dateSpy.reset();
-      window.navigator.mozHour12 = false;
-      f.formatTime(oneTuesday);
-      assert.ok(dateSpy.calledTwice);
-      assert.ok(dateSpy.calledWith(oneTuesday, '%a'));
-      assert.ok(dateSpy.calledWith(oneTuesday,  'shortTimeFormat24'));
-
-      dateSpy.restore();
-      window.navigator.mozHour12 = originalMozHour12;
```

Comment:
I'm assuming you're removing all of this because it's not actually testing a public API. That's great!